README.md

```diff
@@ -1,22 +1,174 @@
-## Example command
+[![Build](https://github.com/canonical/chisel/actions/workflows/build.yml/badge.svg)](https://github.com/canonical/chisel/actions/workflows/build.yml)
+[![Tests](https://github.com/canonical/chisel/actions/workflows/tests.yaml/badge.svg)](https://github.com/canonical/chisel/actions/workflows/tests.yaml)
 
+# Chisel
+
+**Chisel** is a software tool for carving and cutting Debian packages!
+
+It is built on the idea of **Package Slices** - minimal, complimentary and
+loosely coupled sets of files, based on the package’s metadata and content.
+Slices are basically subsets of the Debian packages, with their own content and
+set of dependencies to other internal and external slices.
+
+![pkg-slices](https://canonical-rockcraft.readthedocs-hosted.com/en/latest/_images/package-slices.png)
+
+---
+![a-slice-of-ubuntu](https://canonical-rockcraft.readthedocs-hosted.com/en/latest/_images/slice-of-ubuntu.png)
```

Comment:
This images are broken. Where should the real ones live?

Reply:
Indeed there was a recent change in the rockcraft docs causing this to break. I'm proposing a `docs/_static` folder here to host these images and future expansions of this documentation